test/upstream-e2e-tests.sh

```diff
@@ -48,4 +48,6 @@ if [[ $TEST_KNATIVE_E2E == true ]]; then
   upstream_knative_eventing_e2e
 fi
 
+check_serverless_alerts
```

Comment:
Maybe I could call this function also here in operator-e2e: https://github.com/openshift-knative/serverless-operator/blob/main/test/e2e-tests.sh#L42 . WDYT?